src/NUnitConsole/nunit3-console/TestEventHandler.cs

```diff
@@ -127,6 +128,21 @@ private void WriteLabelLine(string label)
                 _currentLabel = label;
             }
         }
+        private void WriteLabelLine(string label, string resultStatus)
+        {
+            if (label != _currentLabel)
+            {
+                ColorStyle colorByResultStatus = resultStatus == "Passed" ? ColorStyle.Pass : resultStatus == "Failed"
+                            ? ColorStyle.Failure : resultStatus == "Warning"
+                    ? ColorStyle.Warning
+                    : ColorStyle.Output;
```

Comment:
I am going to move the code to separated private method (with switch/case), but probably it should be moved to general "utility/helper"?